ew/cmd/farm/farmcmds.py

```diff
@@ -299,6 +301,7 @@ async def reap(cmd):
 
                         if has_tool and weapon.id_weapon == ewcfg.weapon_id_pitchfork:
                             metallic_crop_ammount *= 2
+                        crop_gain = metallic_crop_ammount * 3
```

Comment:
can't see from the code here, but is this and line 334 mutually exclusive? one overwrites the other before anything is done with the stat. also it kind of looks like you want this inside the if clause at a glance, is that right?

Reply:
crop_gain isn't overwritten by the lower one because this is inside the if statement and line 344 is inside the else statement. crop_gain is set to the amount of crops harvested then multiplied by an xp gain factor at the end.

it doesnt need to be inside that if statement because it gets mutiplied by 3 after metallic_crop_ammount is modified. xp from metallic crop is multiplied by 3 since you get fewer per harvest